taxi/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Car, Manufacturer
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"
+
+
+class ManufacturerForm(forms.ModelForm):
+    class Meta:
+        model = Manufacturer
+        fields = "__all__"

```

Comment:
Form Field Exposure: Similar to the CarForm, using `fields = "__all__"` here can expose sensitive fields. Specify only the necessary fields to include in the form.